web/src/services/Selector.service.ts

```diff
@@ -46,7 +49,7 @@ const getPseudoSelectorString = (pseudoSelector?: TPseudoSelector): string => {
 const SelectorService = () => ({
   getSelectorString(selectorList: TSpanSelector[], pseudoSelector?: TPseudoSelector): string {
     return selectorList.length
-      ? `span[${getFilters(selectorList).join('  ')}]${getPseudoSelectorString(pseudoSelector)}`
```

Comment:
We need the double space to make this work, as we are doing a double space split on line 61